openmc/settings.py

```diff
@@ -49,15 +49,17 @@ class Settings:
         deviation.
     create_fission_neutrons : bool
         Indicate whether fission neutrons should be created or not.
-    cutoff : dict
-        Dictionary defining weight cutoff and energy cutoff. The dictionary may
-        have six keys, 'weight', 'weight_avg', 'energy_neutron', 'energy_photon',
-        'energy_electron', and 'energy_positron'. Value for 'weight'
+    cutoff : dict 
+        Dictionary defining weight cutoff, energy cutoff and time cutoff. The
+        dictionary may have ten keys, 'weight', 'weight_avg', 'energy_neutron',
+        'energy_photon', 'energy_electron', 'energy_positron', 'time_neutron',
+        'time_photon', 'time_electron', and 'time_positron'. Value for 'weight'
         should be a float indicating weight cutoff below which particle undergo
         Russian roulette. Value for 'weight_avg' should be a float indicating
-        weight assigned to particles that are not killed after Russian
-        roulette. Value of energy should be a float indicating energy in eV
-        below which particle type will be killed.
+        weight assigned to particles that are not killed after Russian roulette.
+        Value of energy should be a float indicating energy in eV below which
+        particle type will be killed. Value of time should be a float indicating
+        time in s above which particle type will be killed.
```

Comment:
Same for this documentation: should mention that it is killed before the first collision after the time cutoff.